### PR TITLE
SG-22648: Rebranded to ShotGrid

### PR DESCRIPTION
**Pull request by mathurf**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -293,7 +293,7 @@ def __add_write_node_commands(self, context=None):
                 pn
             )
             self.engine.register_command(
-                "%s [Shotgun]" % profile_name,
+                "%s [ShotGrid]" % profile_name,
```
Comment (jfboismenu) [inline app.py:296]: The command name shouldn't change. If we do the people's scripts may break.
What we should do it keep the same name, but set the title to something else so the right name show's up in the menu.
See documentation for [`register_command`](https://developer.shotgridsoftware.com/tk-core/platform.html#sgtk.platform.Engine.register_command)

File: python/tk_nuke_writenode/handler.py [hunk 2]
```diff
@@ -35,7 +35,7 @@ class TankWriteNodeHandler(object):
     """
 
     SG_WRITE_NODE_CLASS = "WriteTank"
-    SG_WRITE_DEFAULT_NAME = "ShotgunWrite"
+    SG_WRITE_DEFAULT_NAME = "ShotGridWrite"
```
Comment (jfboismenu) [inline python/tk_nuke_writenode/handler.py:38]: Same reasoning here. Could this break compatibility with older scenes?

**Review by jfboismenu — COMMENTED** [review]
I don't feel equipped to give an approval here. Please ask @eshokrgozar or @willis102 to take a look.

File: gizmos/WriteTank.gizmo [hunk 3]
```diff
@@ -33,7 +33,7 @@ Gizmo {
  mapsize {0.15 0.15}
  addUserKnob {
     20 tank_tab
-    l "Shotgun Write"
+    l "ShotGrid Write"
```
[the remark this answers is not in the document]
Comment (mathurf) [reply gizmos/WriteTank.gizmo:36]: This is only for the tab name in Nuke. It is called by WriteTank and should not break anything

File: python/tk_nuke_writenode/handler.py (hunk 2 above)
Comment (mathurf) [reply python/tk_nuke_writenode/handler.py:38]: This should be safe. It is only the default name we give to new writenodes

**Review by jfboismenu — APPROVED** [review]

**Pull request merged**